lib/rules/template-tag-spacing.js

```diff
@@ -0,0 +1,67 @@
+/**
+ * @fileoverview Rule to check spacing between template tags and their literals
+ * @author Jonathan Wilsson
+ */
+
+"use strict";
+
+//------------------------------------------------------------------------------
+// Rule Definition
+//------------------------------------------------------------------------------
+
+module.exports = {
+    meta: {
+        docs: {
+            description: "require or disallow spacing between template tags and their literals",
+            category: "ECMAScript 6",
```

Comment:
I'm not sure, but I think we're generally trying to move stuff out of the "ECMAScript 6" category (also see https://github.com/eslint/eslint.github.io/issues/310). In my opinion, it might be better to put this in "Stylistic Issues".